tests/globalhelper/reporthelper.go

```diff
@@ -141,7 +141,7 @@ func isTestCaseInExpectedStatusInClaimReport(
 	claimReport claim.Root,
 	expectedStatus string) (bool, error) {
 	for testCaseClaim := range claimReport.Claim.Results {
-		if formatTestCaseName(testCaseName) == formatTestCaseName(testCaseClaim) {
+		if strings.Contains(formatTestCaseName(testCaseClaim), formatTestCaseName(testCaseName)) {
```

Comment:
Not sure whether this will work if there are tcs that use the name of another tc as prefix. I don't think we have anything like that, but you should protect agains it.

Reply:
I think that in that case, the tc will have a more unique name that will differentiate it from the other one.
the issue is sometimes tcs on the claim file have a much different name than the actual tc name.

Reply:
TC names should have full match between junit and claim file. @greyerof Correct me if I'm wrong